### ISSUE TITLE
Add currency option to Money field type

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone#474]
It currently formats using `$` which is no good for anyone on a different currency.

While this could be replaced by specifying a different format string, it would be nice to have an explicit currency option that changes the default.

Alternatively, we could implement language support from [numeraljs](http://numeraljs.com) and set / use its global language instead - this is worth looking into.

**Comment by douglasf** [comment]
My solution was to add this to FieldTypes.Money constructor 
```
this._currency = options.currency;
```
and this to the top of the format function
```
	if (this._currency) {
		try {
			numeral.language(this._currency, require('numeral/languages/' + this._currency));
			numeral.language(this._currency);
		} catch (err) {
			throw new Error('FieldType.Money: options.currency failed to load.');
		}
	}
``` 

Then `currency: 'sv-se', format: '0[.]00 $'`would load the sv-se.js from languages folder and format it according to the format option.

Was a very quick patch so I might have overlooked something.

**Comment by morenoh149** [comment]
@douglasf make it a pull request. Looks good.

**Comment by SharadKumar** [comment]
After #984 is this still an open issue?

**Comment by SharadKumar** [comment]
I notice that new option for `currency` in Money field depends on numeraljs languages. But there are hardly enough at the moment: en-us, en-au? And it doesn't allow the case when a `culture` has multiple currencies, since you are configuring a culture not currency.
https://github.com/adamwdraper/Numeral-js/tree/master/languages

**Comment by creynders** [comment]
I I'm not mistaken this was added with 15c4b69

**Comment by johnlo1992** [comment]
looks like the new numeral update broke the old functionality, by renaming "language" to "locale"

`if (this.currency) {
		try {
			numeral.locale(this.currency, require('numeral/locales/' + this.currency));
			numeral.locale(this.currency);
		} catch (err) {
			throw new Error('FieldType.Money: options.currency failed to load.');
		}
	}`

**Comment by adsp16** [comment]
Still cant use pounds as a format string, only defaults to $
